tests/jenkins/nrnivmodl.sh

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/bash
 
 set -e
-set +x
+set -x
 
 TEST_DIR="$1"
 
 . /gpfs/bbp.cscs.ch/apps/hpc/jenkins/config/modules.sh
-module load neuron/2018-10/python3/parallel intel
+module load intel
+
+neuron_version=$(module av neuron 2>&1 | grep -o -m 1 '^neuron.*/parallel$' | awk -F' ' '{print $1}')
+if [[ $neuron_version ]]; then
+    module load $neuron_version
+else
+    echo "No compatible neuron version found."
+    exit 1
+fi
 
```

Comment:
Here we can do `module list` so that we see what's loaded.

Reply:
I added this in both cases to check what is loaded in any case